### PR TITLE
feat: implement new website homepage

### PR DESCRIPTION
**Pull request by gui-santos**
# Purpose of PR

Screenshot 2022-03-22 at 11 18 14

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/main/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

**Comment by mshaaban0** [comment]
My 2 cents on this that it could use some colors, maybe icons on cards or something. It kinda look cheap 🥲

**Comment by domarku** [comment]
Alright friends, thanks for the feedback! What I'm getting is "back to the drawing board". It's not yet ready for a PR. Let me iterate on it a bit and get more of your feedback on the design before we put it up :)

File: packages/website/pages/index.tsx [hunk 1]
```diff
@@ -1,105 +1,95 @@
...
+      <Flex className={styles.sections} gap="spacing2Xl">
+        <Flex flexDirection="column" alignItems="flex-start">
+          <Image src={figmaSVG} alt="Figma’s logo" />
+
+          <Heading marginTop="spacingM">Figma UI Kit</Heading>
+          <Paragraph>
+            Copy the UI Kit to Figma, publish it as a Team library and start
+            prototyping.
+          </Paragraph>
+          <TextLink href="https://www.figma.com/" target="_blank">
```
Comment (domarku) [inline packages/website/pages/index.tsx:76]: @gui-santos here's the real link:
https://www.figma.com/@contentful

**Review by domarku — APPROVED** [review]

**Comment by domarku** [comment]
@gui-santos let's wait for Monday to merge the homepage. I need to prepare the UI Kit for publishing today.

**Pull request merged**